app/components/common/blankGraph/blankGraph.html

```diff
@@ -0,0 +1,6 @@
+<article class="cfa-contentBlock--singleItem">
+	<h3 class="cfa-blankGraph-title">{{vm.graphTitle}}</h3>
+	<div class="cfa-blankGraph-content">
+		<div ng-transclude></div>
+	</div>
+</article>
```

Comment:
bc it's a self-containing element with its own header. I've been going back and forth on it, but my thinking for leaving it was bc _technically_ I believe it's the right html structure, which will help with indexing when we start to worry about SEO...thoughts?